endpoints/openrtb2/auction.go

```diff
@@ -444,6 +444,10 @@ func fillAndValidateNative(n *openrtb.Native, impIndex int) error {
 }
 
 func validateNativeContextTypes(cType native.ContextType, cSubtype native.ContextSubType, impIndex int) error {
+	if cType == 0 {
```

Comment:
Can we please add a unit test or json test for this case?

Reply:
endpoints/openrtb2/sample-requests/valid-native/request-no-context.json

Reply:
Ah. Changing the directory changes the validation criteria. Missed that.